Releases.md

```diff
@@ -27,6 +28,39 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110)   | [DML_FEATURE_LEVEL_2_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_2_0) | Windows 10 May 2020 Update, Version 2004 (Build 10.0.19041, 20H1) | -                                                                                           |
 | [1.0.0](#directml-100)   | [DML_FEATURE_LEVEL_1_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_1_0) | Windows 10 May 2019 Update, Version 1903 (Build 10.0.18362; 19H1) | -                                                                                           |
 
+# DirectML 1.15.0
+- Introduced DML_FEATURE_LEVEL 6.4:
+  - Added 3 new operators:
+    - DML_OPERATOR_RESAMPLE3
+    - DML_OPERATOR_FOLD
+    - DML_OPERATOR_UNFOLD
+  - Updated 3 operators:
+    - DML_OPERATOR_PADDING (updated to accept DML_PADDING_MODE::DML_PADDING_MODE_WRAP)
+    - DML_OPERATOR_PADDING1 (updated to accept DML_PADDING_MODE::DML_PADDING_MODE_WRAP)
+    - DML_OPERATOR_ACTIVATION_SOFTPLUS (updated to allow Steepness < 1)
+
+- Introduced DML_FEATURE_LEVEL 6.3:
+  - Added data type Support:
+    - DML_TENSOR_DATA_TYPE_UINT4
+    - DML_TENSOR_DATA_TYPE_INT4
+  - Added 4 new operators:
+    - DML_OPERATOR_MEAN_VARIANCE_NORMALIZATION2
+    - DML_OPERATOR_MULTIHEAD_ATTENTION1
+    - DML_OPERATOR_QUANTIZE (added with INT4/UINT4 support)
+    - DML_OPERATOR_DEQUANTIZE (added with INT4 support)
```

Comment:
```suggestion
    - DML_OPERATOR_DEQUANTIZE (added with INT4/UINT4 support)
```
Consistent with QUANTIZE (I'm seeing Dequantize supports ["float16", "float32", "uint4", "int4", "uint8", "int8"], but it's okay if we only mention int4).

Reply:
I updated to mention int4/uint4 as specific callouts.